ethereum_history_api/circuits/lib/src/account_with_storage_int_test.nr

```diff
@@ -50,3 +52,13 @@ fn test_get_account_with_storage_invalid_state_root() {
     let storage_key = proofs[0].key;
     let _ = get_account_with_storage(number, address, storage_key);
 }
+
+#[test(should_fail)]
+fn test_get_account_with_storage_invalid_storage_root() {
+    let _ = OracleMock::mock("get_header").returns((block_header_partial_from_different_block, block_header_rlp_from_different_block));
+    let _ = OracleMock::mock("get_proof").returns((account_from_different_block, state_proof_from_different_block, proofs[0]));
```

Comment:
Wouldn't it be more readable if we use all variables from usdc and just proof from cyber_punks?